tests/test_callprofiler.py

```diff
@@ -380,4 +380,44 @@ def foo():
         modmesh.call_profiler.reset()
         foo()
 
+    def test_status(self):
+        modmesh.wrapper_profiler_status.disable()
+        self.assertFalse(modmesh.wrapper_profiler_status.enabled)
+
+        modmesh.call_profiler.reset()
+        buf = modmesh.ConcreteBuffer(10)
+        self.assertEqual({}, modmesh.call_profiler.result())
+
+        modmesh.wrapper_profiler_status.enable()
+        modmesh.call_profiler.reset()
+        buf = modmesh.ConcreteBuffer(10)  # noqa: F841
+        res = modmesh.call_profiler.result().get("children")
+        self.assertEqual(['ConcreteBuffer.__init__'], [d["name"] for d in res])
+
+    def test_names(self):
+        modmesh.call_profiler.reset()
+        buf = modmesh.ConcreteBuffer(10)
+        buf2 = buf.clone()  # noqa: F841
```

Comment:
Don't change the testing code style.  The original style was:
```python
        buf2 = buf.clone()  # noqa: F841
        self.assertEqual(
            ['ConcreteBuffer.__init__', 'ConcreteBuffer.clone'],
            modmesh.time_registry.names)
```

Creating the new variable `name` changes the style unnecessarily.  Keep the style and make it like:
```python
        buf2 = buf.clone()  # noqa: F841
        res = modmesh.call_profiler.result().get("children")
        self.assertEqual(
            ['ConcreteBuffer.__init__', 'ConcreteBuffer.clone'],
            [d['name'] for d in res])
```